common/lib/xmodule/xmodule/modulestore/xml.py

```diff
@@ -46,8 +47,17 @@
 # TODO (cpennington): Remove this once all fall 2012 courses have been imported
 # into the cms from xml
 def clean_out_mako_templating(xml_string):
+    orig_xml = xml_string
     xml_string = xml_string.replace('%include', 'include')
     xml_string = re.sub(r"(?m)^\s*%.*$", '', xml_string)
+    if orig_xml != xml_string:
+        dog_stats_api.increment(
+            'vscompat.deprecation',
+            tags=(
+                "string:{}".format(orig_xml),
```

Comment:
Is this `string` going to have a relatively small number of different values? It seems like not, so I wouldn't want to put it in the datadog tag.